tools/ipptransform.c

```diff
@@ -1850,9 +1846,13 @@ xform_document(
 	      if (!(page & 1) && ras.header.Duplex)
 		CGContextConcatCTM(context, back_transform);
 	      CGContextConcatCTM(context, transform);
-
-	      CGContextClipToRect(context, CGPDFPageGetBoxRect(pdf_page, kCGPDFCropBox));
-	      CGContextDrawPDFPage(context, pdf_page);
+          CGContextClipToRect(context, renderer.getPageRect(rendering));
+          CGContextSetRGBFillColor(context, 0.8, 0.8, 0.8, 1.);
```

Comment:
What is this color we're setting here?

Reply:
Good catch. Left over from an experiment.